scripts/update.sh

```diff
@@ -16,6 +16,11 @@ case $key in
     echo "$USAGE" >&2
     exit 1
     ;;
+    --stack-name)
+    STACKNAME="$2"
+    shift # past argument
```

Comment:
nitpick: looks like redundant comments to me, but I guess you put them there to be consistent with the other arguments. We can address them in another PR! :)